src/components/Account/Account.tsx

```diff
@@ -1,8 +1,13 @@
-import React, { useMemo } from 'react';
-import { useTranslation } from 'react-i18next';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
 import { useLocation, useNavigate } from 'react-router-dom';
 import shallow from 'zustand/shallow';
 import DOMPurify from 'dompurify';
+import { useMutation } from 'react-query';
+
+import Alert from '../Alert/Alert';
```

Comment:
Please use absolute paths (with '#' aliases) instead of relative.
```suggestion
import Alert from '#components/Alert/Alert';
```